docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/element-tracking/index.md

```diff
@@ -691,9 +691,7 @@ These events include an `element` entity which includes:
 - Size and position information
 - Any attributes collected via the [`detail` setting](#element-data)
 
-```json reference title="Entity - iglu:com.snowplowanalytics.snowplow/element/jsonschema/1-0-0" referenceLinkText="See schema on GitHub"
-https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/element/jsonschema/1-0-0
-```
+[See `element` entity schema on GitHub](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/element/jsonschema/1-0-0)
```

Comment:
Nit: we also have the `SchemaProperties` component that can show a collapsible with the schema ([see here](https://github.com/snowplow/documentation/blob/main/docs/events/ootb-data/app-information/index.md?plain=1#L52-L58)), but not sure if it fits here.

Reply:
Neat! I was kind of hoping to not have to hardcode anything, and that seems to require a copy of the entire schema. :( Think I'll leave the links for now.